lib/internal/once.js

```diff
@@ -1,8 +1,10 @@
 export default function once(fn) {
-    return function (...args) {
+    function wrapper (...args) {
         if (fn === null) return;
         var callFn = fn;
         fn = null;
         callFn.apply(this, args);
-    };
+    }
+    Object.assign(wrapper, fn)
```

Comment:
What is the purpose of this change?

Reply:
So static properties attached to the `fn` get attached to the `wrapper`, namely the `PROMISE_SYMBOL` attached to `promiseCallback`s.